app/src/main/res/values/strings.xml

```diff
@@ -1,121 +1,121 @@
-    <string name="ccontinue">Continue</string>
-    <string name="disconnect_from_library">Disconnect</string>
-    <string name="your_bookmarks_will_be_kept">Your bookmarks will be kept</string>
-    <string name="template_error_failed_to_disconnect">Failed to disconnect: %1$s</string>
-    <string name="play_video">Play</string>
-    <string name="pause_video">Pause</string>
-    <string name="no_media_found">No media found</string>
-    <string name="view_media">View</string>
-    <string name="open_in_web_viewer">Open in web viewer</string>
-    <string name="albums">Albums</string>
-    <string name="loading_albums_progress">Loading albums…</string>
-    <string name="reload_albums">Reload albums</string>
-    <string name="no_albums_found">No albums found</string>
-    <string name="clear_selection">Clear selection</string>
-    <string name="done_selecting">Done</string>
+    <string name="export_bookmarks">εξαγωγή σελιδοδείκτών</string>
+    <string name="import_bookmarks">εισαγωγή σελιδοδειτών</string>
+    <string name="template_error_failed_to_import_bookmarks">αδυναμία εισαγωγής σελιδοδεικτών:: %1$s</string>
+    <string name="template_successfully_imported_bookmarks_plural">επιτυχείς εισαγωγή %1$d %2$s</string>
+    <string name="replace_current_bookmarks_with_saved">αντικατασταση των υπαρχουσών σελιδοδεικτών με τους αποθηκευμένους</string>
+    <string name="save_bookmarks_to_file">αποθήκευση σελιδοδεικτών σε αρχείο</string>
+    <string name="your_bookmarks_will_be_replaced">οι υπάρχων σελιδοδείκτες θα αντικατασταθούν. συνέχεια?</string>
+    <string name="ccontinue">συνέχεια</string>
+    <string name="disconnect_from_library">αποσύνδεση</string>
+    <string name="your_bookmarks_will_be_kept">οινσελδοδεικτες σου θα διατηρηθούν</string>
+    <string name="template_error_failed_to_disconnect">αποτυχία αποσύνδεσης: %1$s</string>
+    <string name="play_video">αναπαραγωγή</string>
+    <string name="pause_video">παύση</string>
+    <string name="no_media_found">δε βρέθηκαν πολυμέσα</string>
+    <string name="view_media">προβολή</string>
+    <string name="open_in_web_viewer">άνοιγμα σε περιηγητή</string>
+    <string name="albums">άλμπουμ</string>
+    <string name="loading_albums_progress">φόρτωση άλμπουμ…</string>
+    <string name="reload_albums">επαναφόρτωση άλμπουμ</string>
+    <string name="no_albums_found">δε βρέθηκαν άλμπουμ</string>
+    <string name="clear_selection">καθαρισμος επιλογής</string>
+    <string name="done_selecting">ολοκλήρωση</string>
     <string name="template_current_of">%1$d of %2$d</string>
-    <string name="search_doesnt_fit_allowed_types">Search query doesn\'t fit the allowed types</string>
-    <string name="save_file_to_the_storage">Save the file to the storage</string>
-    <string name="share_the_file">Share the file</string>
+    <string name="search_doesnt_fit_allowed_types">ο όρος αναζήτησης δεν\ταυτίζεται με τους επιτρεπόμενους τύπους</string>
```

Comment:
The `\` in the strings is for escaping `'`. If there is no `'` or other XML special symbols in your string, you can remove `\` as well.